content-services/latest/install/zip/additions.md

```diff
@@ -4,31 +4,6 @@ title: Install additional software
 
 You can install the third-party software used by Content Services independently. Some of the software can be installed any time before or after installing Content Services.
```

Comment:
I guess this wont show? Its not red !

Reply:
Haven't touched anything in zip installs...